starcounter-include.html

```diff
@@ -15,15 +15,47 @@
         function warnAboutDeprecatedPartial() {
             console.warn('`partial` attribute and property are deprecated from `starcounter-include` in favour of (viewModel property or view-model attribute), they will soon be no longer be supported');
         }
+        const BLOCKING_LINKS_SELECTOR = 'link[rel="stylesheet"]:not([async])';
+
         const isWebkit = navigator.vendor && navigator.vendor.indexOf("Apple") > -1;
 
         // still needed for .selectNode
         const isSafari = isWebkit && navigator.userAgent && !navigator.userAgent.match("CriOS");
         const fallBackComposition = '<style>:host{display:block;}</style><slot></slot>';
 
+        /**
+         * Returns a rejectable promise to load (or fail to load) all blocking
+         * link elements in given scope.
+         * @param  {Node} scope scope to be queried for the elements
+         * @return {Promise|null} `.reject()`able promise that all links will be loaded,
+         *                        or `null` if there are none
+         */
+        function blockingLinksLoaded(scope){
+            const links = scope.querySelectorAll(BLOCKING_LINKS_SELECTOR);
+            if(links.length === 0){
+                return null;
+            }
+            let rejectDefer;
+            const promise = new Promise((resolveAll, rejectAll) => {
+                Promise.all(Array.from(links).map((link)=>{
+                    return new Promise((resolve, reject)=>{
+                        if(link.sheet){
+                            resolve();
+                        } else {
+                            link.addEventListener('load', resolve);
+                            link.addEventListener('error', resolve);
+                        }
+                    });
+                })).then(resolveAll);
+                rejectDefer = rejectAll;
+            });
+            promise.reject = rejectDefer;
```

Comment:
Can you rename this prop to `cancel`? Because currently it's tangled with Promise's typical `reject` while it's a bit different. Anything not `reject` would do.

Reply:
The similarity to typical `Promise.reject` was on purpose, as it's somewhat common standard to call it that way for defered object:
 - https://developer.mozilla.org/en-US/docs/Mozilla/JavaScript_code_modules/Promise.jsm/Deferred#backwards_forwards_compatible
 - https://api.jquery.com/jquery.deferred/

Reply:
I recognize this pattern and I think that the name `.reject` is fine. 

Another info about this pattern: https://stackoverflow.com/questions/26150232/resolve-javascript-promise-outside-function-scope